qa/rpc-tests/wallet_import_export.py

```diff
@@ -18,6 +18,9 @@ def run_test(self):
         # node 0 should have the keys
         dump_path0 = self.nodes[0].z_exportwallet('walletdump')
         (t_keys0, sprout_keys0, sapling_keys0) = parse_wallet_file(dump_path0)
+
+        for sapling_key0 in sapling_keys0.splitlines():
+            assert_equal(4, len(sapling_key0.split(' #')[0].split()), "Should have 4 parameters before ' #'")
```

Comment:
We should test importing a key with `z_importkey` before exporting with `z_exportwallet`; those keys should only have two parameters.